viewing_party/party.py

```diff
@@ -1,23 +1,185 @@
 # ------------- WAVE 1 --------------------
-
+#---Wave_1_function_1---
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
+    movie_dict = {}
+
+    if not title:
+        return None
+    elif not genre:
+        return None
+    elif not rating:
+        return None
+    else: 
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+
+    return movie_dict
+
+create_movie("Title", "Genre", "Rating")
+
+
+# #---Wave_1_function_2---
+def add_to_watched(user_data, movie):
+
+    user_data["watched"].append(movie)
+    return user_data
+
+add_to_watched({"watched": []}, {
+        "title": "MOVIE_TITLE_1",
+        "genre": "GENRE_1",
+        "rating": "RATING_1"})
+
+
+# #---Wave_1_function_3---
+def add_to_watchlist(user_data, movie):
+
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+
+#---Wave_1_function_4---
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title != (movie["title"]):
+            continue
+        else:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+            print (user_data)

```

Comment:
[nit] Flagging a stray `print` statement that got left behind. Also, lines 20 and 29-32. Keep printing things and testing them! It'd be nice if they were deleted by submission time, or even before they're committed to git.